### PR TITLE
starter player and room

### PR DESCRIPTION
**Pull request by yoshimii**

### CODE REVIEW

File: src/adv.py [hunk 1]
```diff
@@ -33,19 +44,180 @@
...
 # * Waits for user input and decides what to do.
-#
 # If the user enters a cardinal direction, attempt to move to the room there.
 # Print an error message if the movement isn't allowed.
-#
 # If the user enters "q", quit the game.
+
+def moveDirection(direction):
+    """A helper function that changes the location of the player."""
+    if direction == 'north' and hasattr(room[player_info.location], 'n_to') and (room[player_info.location].n_to != room[player_info.location]):
```
Comment (tsaihuangsd) [inline src/adv.py:86]: when you have time, try to make your repl more flexible in the user input (e.g. 'n' or "N" also can be accepted for 'north').  

Not too clear about the second condition of your if statement...  Care to explain with some comments?

File: src/adv.py [hunk 2]
```diff
@@ -33,19 +44,180 @@
...
+            # cur_rm_desc = room[player_info.location]
+            to_rm_desc = room[player_info.location].n_to
+            for i in room.keys():                
+                if player_info.location == i and hasattr(room[player_info.location], 'n_to'):                    
+                    for i in room.keys():
+                        if to_rm_desc == room[i]:
+                            player_info.location = i
+        print('Current room: ', player_info.location)
+        print(f'You are currently in {player_info.location}:\n\nRoom description:\n{room[player_info.location]}\n\n')
+    elif direction == 'south' and hasattr(room[player_info.location], 's_to') and (room[player_info.location].s_to != room[player_info.location]):
```
Comment (tsaihuangsd) [inline src/adv.py:97]: A lot of your codes for south, east, and west moves can be reusing codes for north move.  Try to make this dryer when you get a chance

File: src/adv.py [hunk 3]
```diff
@@ -33,19 +44,180 @@
...
+        print('Oof dusty.')
+    else:
+        print('The horrors you see are: ')
+        for i in room[player_info.location].items:
+            print(i) 
+
+def take():
+    if len(room[player_info.location].items) == 0:
+        print("Room's been cleaned out. By sombody... \n or something! Or was it me? <<hiccup>>")
+    else:
```
Comment (tsaihuangsd) [inline src/adv.py:144]: Try to give the player an option to pick items one at a time, especially if a game sets a limit to the number of items a player can take at any given time

File: src/adv.py [hunk 4]
```diff
@@ -33,19 +44,180 @@
...
+        
+if __name__ == '__main__':
+    print('Space Survival Horror Game!')
+    print('===========================')
+    print()
+    print('Game Controls:')
+    print('north south east west')
+    print()
+    print('(Type "help" for commands.)')
+    GameControls().cmdloop()
```
Comment (tsaihuangsd) [inline src/adv.py:236]: Interesting...  I did not know about cmd.cmdloop()!  Great job!

File: src/adv.py [hunk 5]
```diff
@@ -33,19 +44,180 @@
...
+
+def check_bag():
+    if len(player_info.items) == 0:
+        print('nothing in here yet\nkeep your eyes peeled')
+    else:
+        print('Your items:')
+        for i in player_info.items:
+            print(i)
+    
+def drop():
```
Comment (tsaihuangsd) [inline src/adv.py:157]: I think there is a bug in the drop function still.  It only display rucksack after I picked up some items and the program crashes when I enter '2'

**Review by tsaihuangsd — COMMENTED** [review]
Although there are some opportunities to make your code dryer, but overall, excellent job!!

File: src/adv.py (hunk 5 above)
Comment (yoshimii) [reply src/adv.py:157]: bug fixed!

**Pull request merged**